producer-kafka/README.adoc

```diff
@@ -0,0 +1,407 @@
+= Using the Kafka CDC Producer
+
+This extension provides support for sending Change Data Capture(CDC) events to an Apache Kafka®.
+It is designed to work with the Stargate.
+
+== Kafka CDC - Stargate integration
+
+This extension is designed to integrate with Stargate. It uses an link:https://www.osgi.org/[OSGi] framework for registering the extension within Stargate.
+The `KafkaProducerActivator` class handles the registration process. When you put the `producer-kafka-{version}.jar` in the
+`/stargate-lib` directory it will be automatically picked, and the activator logic is invoked.
+
+This extension has one service required: `Metrics` - this is provided by the `CoreActivator` in the link:../core[core] module.
+
+== Architecture
+
+The main entry point for this CDC connector is a link:../producer-kafka/src/main/java/io/stargate/producer/kafka/KafkaCDCProducer.java[KafkaCDCProducer].
+It implements the SchemaAwareCDCProducer interface that serves as an integration layer between Stargate's persistence and CDC extension.
+
+1. `init()` - takes the map of configuration settings that are passed from Stargate to the CDC connector. The configuration settings are described in <<Configuration>>
+
+2. `createTableSchemaAsync()` - MUST be invoked on every Stargate node before any call to `send` is made (including restarts).
```

Comment:
> MUST be invoked

I think it sounds like the end user needs to do something here. We could state that it _will_ be invoked.

Reply:
This is documentation from the perspective of the CDC particular producer. So the extension does not influence that it _will_ be invoked. I try to signal that it MUST be invoked, and this extension's logic relies on that fact.